framework/src/org/checkerframework/framework/type/AnnotatedTypeFactory.java

```diff
@@ -529,32 +537,142 @@ public TypeArgumentInference getTypeArgumentInference() {
     }
 
     /**
-     * If the type factory or checker class is annotated with {@link
-     * TypeQualifiers}, return an immutable set with the same set
-     * of classes as the annotation.  If the class is not so annotated,
-     * return an empty set.
+     * Returns an immutable set of annotation classes that are supported by a checker
```

Comment:
I think this is completely copied from the manual.  I would just link to the manual chapter so that we don't have to update the documentation in two places if it changes:
@checker_framework.manual #indicating-supported-annotations Indicating supported annotations

(An alternative would be to link to the JavaDoc in the manual chapter.)